src/Components/VitalsMonitor/types.ts

```diff
@@ -46,6 +46,7 @@ export interface ChannelOptions {
 }
 
 export interface IVitalsComponentProps {
+  minutesSinceAssignment?: number;
```

Comment:
1. This variable name is confusing. Another contributor may not understand what this prop does. `minutesSinceAssignment` of what?

2. Consider passing the `datetime` before it should hide rather than passing minutesSinceAssignment. Since vitals monitor component maybe kept visible for prolonged period of time, the `minutesSinceAssignment` needs to change over time as minutes passes. Hence passing the actual cut-off datetime itself would be better.